tapdb/sqlc/queries/universe.sql

```diff
@@ -216,8 +216,17 @@ WITH asset_supply AS (
     JOIN genesis_info_view gen
         ON leaves.asset_genesis_id = gen.gen_asset_id
     GROUP BY gen.asset_id
+), group_supply AS (
+    SELECT sum AS num_assets, uroots.group_key AS group_key
```

Comment:
IIUC, sum here is the amount issued, not the number of assets.

Reply:
If we want it to be a sum, then we actually need to do an aggregation value also.

Reply:
Nvm this is fine, just name threw me off.